views/components/Graphs/BarGraph.jsx

```diff
@@ -28,26 +38,32 @@ const BarGraph = ({ graph }) => {
         labelComponent={<VictoryLabel />}
       />
       <VictoryAxis label='Site' style={graphStyles.xAxis} />
-      <VictoryAxis label='Total' dependentAxis style={graphStyles.yAxis} />
+      <VictoryAxis
+        label='Total'
+        dependentAxis
+        style={graphStyles.yAxis}
+        tickFormat={(yAxisValue) => `${yAxisValue}%`}
+      />
       <VictoryStack>
         {Object.values(graph.data).map((data, idx) => (
           <VictoryBar
             data={data}
             x='study'
-            y='count'
+            y='percent'
             key={'bar' + idx}
             style={{
               data: {
                 fill: ({ datum }) => datum.color,
               },
             }}
-            labels={({ datum: { count, studyTarget, study } }) =>
-              `Study: ${study} \n Current: ${count} \n Target: ${studyTarget}`
+            labels={({ datum }) =>
+              !!datum.percent ? `${datum?.percent}%` : null
             }
             labelComponent={
-              <VictoryTooltip
-                constrainToVisibleArea
-                style={{ fill: ({ datum }) => datum.color }}
+              <VictoryLabel
+                dy={15}
+                labelPlacement='perpendicular'
+                style={{ fill: '#EDEFFA', fontSize: 8 }}
```

Comment:
do we have a colors object or constant we could put this hexcode in? (also why is it no longer coming from datum?)

Reply:
Yeah good point about the constant. The datum object reflects the graph data information, the label styling is separate from the bar graph return object.

Reply:
🎱 might be worth grouping these constants by "purpose" (`colors.anti_flash_white`), and/or exporting a "theme" object ( `theme.colors.anti_flash_white` & `theme.fontSizes.medium` (8) | `theme.fontSizes.large` (12) _for example_) )
```suggestion
export const colors = {
  dark_sky_blue: '#97C0CE',
  neutral_blue: '#5790BD',
  anti_flash_white: '#EDEFFA',
}
```

Reply:
Thanks for the themes idea. For now the colors have been put into an object because there are meetings around a new design for the app and I assume there will be some changes that will have to be implemented.

https://github.com/AMP-SCZ/dpdash/pull/125/commits/3b0ceef3445493543ede7fa1c16e9db8639f83fd